### ISSUE TITLE
Being able to manage the scan site list

### ISSUE THREAD

**Issue by Dostoi** [raulhaag/MiMangaNu#278]
So I'm going to ask for this again ^^

Is it possible to add a way to manage the scan list that is shown when you click on the cross.
I'm only using 2 of them and it's not great to have to search for them every time.

**Comment by xtj-9182** [comment]
To clarify you mean the server list ? currently we have this ServerBase Array we could probably change that into an ArrayList to make removing and adding easy however we would probably need an extra UI which sounds like work :0. The most "ghetto" low effort UI we could do would be to add all servers to the PreferenceFragment as check boxes but that would be really ghetto and low effort though...

**Comment by Dostoi** [comment]
I mean the list that's shown when you click on the arrow on the main menu, that show all the scan site where you can find scan.
I'd like to be able to select which one I want to see, to avoid searching for too long for one

**Comment by xtj-9182** [comment]
yeah the server list, with sites like mangapanda, mangareader etc. I think we're talking about the same thing.

**Comment by raulhaag** [comment]
may be last used on top? and new ones

**Comment by Dostoi** [comment]
I was more thinking about only seeing those I'm using but it would do the same thing in the end

**Comment by raulhaag** [comment]
![editservelist](https://cloud.githubusercontent.com/assets/2048339/15993905/9fe1dc78-30ca-11e6-9cb6-76f5f4badee6.jpg)
cad to edit server list added

**Comment by Dostoi** [comment]
Thanks a lot ! :)

Just a small downside, it's hard to know that you can manage the list. Maybe adding an option in the parameter menu to access this feature

**Comment by raulhaag** [comment]
added menu to acces edit mode :-D

**Comment by raulhaag** [comment]
@Dostoi can you correct french translation before to release next version?

**Comment by Dostoi** [comment]
I'm on it :)

**Comment by Dostoi** [comment]
Should be good

**Comment by Dostoi** [comment]
An issue with the new option in the menu.
If you select it then use the back button of the phone, it show the list you're currently using instead of closing it

**Comment by raulhaag** [comment]
@Dostoi test last version to see if i undertand the problem

**Comment by Dostoi** [comment]
Perfect ! :)

Maybe one last little thing but it's not important ^^
If you have less server in the used server list than in the predefined list, you still have a huge window which is mostly empty in my case.
Maybe having an adaptive window size for the used server list to avoid that.

![screenshot_2016-06-19-08-48-02](https://cloud.githubusercontent.com/assets/7694864/16175803/1728c3d4-35fb-11e6-84b5-18bdc86de870.png)

**Comment by Zykino** [comment]
A popup too short may be not seen instantly when on a big screen.
Your snapshot is acceptable for me. (If the background was not plain gray but had some sort of decoration it could be even better)
